net/isc-dhcp/files/etc/config/dhcrelay

```diff
@@ -7,6 +7,8 @@ config dhcrelay ipv4
 
 	# network interfaces to listen on (e.g. lan or wan)
 	option 'interfaces' ''
+	#option 'upstream_interfaces' ''
```

Comment:
Why does this option need to be commented out for an empty list?

Wouldn't an empty list result in zero iterations through the `for` loop?

Reply:
[ Oops.  Just noticed this needed my attention. ]

Indeed.  It could just be empty and uncommented out.